sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/ExternalTypescriptSensor.java

```diff
@@ -328,6 +339,37 @@ private void saveHighlights(SensorContext sensorContext, Highlight[] highlights,
     highlighting.save();
   }
 
+
+  @Nullable
+  private static File getTypescriptLocation(File topDir) {
+    File node_modules = getDir(topDir, "node_modules");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00117)